examples/boundingbox_camera/Main.cc

```diff
@@ -271,6 +272,16 @@ int main(int _argc, char** _argv)
     }
   }
 
+#ifdef __APPLE__
+  GraphicsAPI graphicsApi = GraphicsAPI::METAL;
+#else
+  GraphicsAPI graphicsApi = GraphicsAPI::OPENGL;
+#endif
+  if (_argc > 2)
+  {
+    graphicsApi = GraphicsAPIUtils::Set(std::string(_argv[2]));
+  }
+
```

Comment:
Can we refactor this out into a utility function so that it would only have to be updated in one place?

Reply:
yep done, added a convenience function `defaultGraphicsAPI()`, [522e36c](https://github.com/gazebosim/gz-rendering/pull/937/commits/522e36c9dc2ffbd7a76feeb3db5cf34854762626)